### ISSUE TITLE
Flutter camera plugin record quality is very poor

### ISSUE THREAD

**Issue by divyanshub024** [flutter/flutter#26959]
## Description

I'm using flutter [camera plugin](https://pub.dartlang.org/packages/camera) and the recording is working fine. But after that when I play the recording the video quality is very poor.  
## Logs

```
```

```
Divyanshus-MacBook-Pro:~ divyanshub024$ flutter doctor -v
[✓] Flutter (Channel stable, v1.0.0, on Mac OS X 10.14.2 18C54, locale en-IN)
    • Flutter version 1.0.0 at /Users/divyanshub024/Softwares/flutter
    • Framework revision 5391447fae (8 weeks ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[!] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/divyanshub024/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/divyanshub024/Library/Android/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1136-b06)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 1.9.4
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.2)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.1.1
    • Dart plugin version 181.5656
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1136-b06)

[✓] IntelliJ IDEA Community Edition (version 2018.3.3)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 31.3.4
    • Dart plugin version 183.5153.38

[✓] Connected device (1 available)
    • ASUS X00TD • J7AAGF01H563MZA • android-arm64 • Android 8.1.0 (API 27)

! Doctor found issues in 1 category.
Divyanshus-MacBook-Pro:~ divyanshub024$ 
```

**Comment by divyanshub024** [comment]
@zoechi Is there anything that can be done to improve the quality right now?

**Comment by zoechi** [comment]
I don't have deep knowledge of the plugin. You could try on StackOverflow or debug the plugin.

**Comment by divyanshub024** [comment]
@zoechi I have figured out something. The video encoding bit rate is very low `mediaRecorder.setVideoEncodingBitRate(1024 * 1000)`. If you increase it to 3000000 then it work's fine.

**Comment by zoechi** [comment]
Pull Requests are welcome ;-)

**Comment by divyanshub024** [comment]
@zoechi Sure, Thanks

**Comment by divyanshub024** [comment]
https://github.com/flutter/plugins/pull/1107

**Comment by quentinleguennec** [comment]
Hi @divyanshub024 . I added a way to control video and picture capture quality.
You can check the PR here:
flutter/plugins#1403
Suggestions are welcome. Don't hesitate to leave a comment to push the review process if that works for you.

**Comment by mvanbeusekom** [comment]
Closing this issue as this seems to be solved with PR flutter/plugins#1952 allowing users to set different quality presets.

If I overlooked something and this is still an issue please leave a comment and I would be happy to reopen the issue.
